test/unit/paging.ts

```diff
@@ -1,11 +1,27 @@
-import { Events } from '@storefront/core';
+import { Events, Selectors } from '@storefront/core';
 import Paging from '../../src/paging';
 import suite from './_suite';
 
-suite('Paging', ({ expect, spy, itShouldBeConfigurable, itShouldHaveAlias }) => {
+suite('Paging', ({ expect, spy, stub, itShouldBeConfigurable, itShouldHaveAlias }) => {
   let paging: Paging;
-
-  beforeEach(() => paging = new Paging());
+  const page = <any>{
+    sizes: {
+      items: [10, 20, 30],
+      selected: 0
+    },
+    current: 5,
+    previous: 4,
+    next: 6,
+    last: 10,
+    from: 41,
+    to: 50
+  };
+
+  beforeEach(() => {
+    paging = new Paging();
+    const select = paging.select = stub();
```

Comment:
Make select a suite level variable along with paging, so it can be altered per test if needed.